ui/src/plugin/PluginDetailView.tsx

```diff
@@ -0,0 +1,259 @@
+                    fullWidth={true}
+                    disabled={
+                        !this.state.unsavedChanges ||
+                        this.state.unsavedChanges === this.props.initialConfig
+                    }
+                    className="config-save"
+                    onClick={() => {
+                        let newConfig = ((this.refs.monaco as any).editor as any) // tslint:disable-line
+                            .getModel()
+                            .getValue();
+                        this.props
+                            .save(newConfig)
+                            .then(() => {
+                                this.setState({unsavedChanges: ''});
+                            })
+                            .catch((err) => {
+                                console.error(err);
+                            });
+                    }}>
+                    Save
+                </Button>
+            </div>
+        );
+    }
+}
+
+class DisplayerPanel extends Component<{pluginInfo: IPlugin; displayText: string}> {
+    public render() {
+        return <ReactMarkDown source={this.props.displayText} />;
+    }
+}
+class PluginInfo extends Component<{pluginInfo: IPlugin}> {
+    public render() {
+        return (
+            <div>
+                {this.props.pluginInfo.name ? (
+                    <p className="name">
+                        Name: <span>{this.props.pluginInfo.name}</span>
+                    </p>
+                ) : null}
+                {this.props.pluginInfo.author ? (
+                    <p className="author">
+                        Author: <span>{this.props.pluginInfo.author}</span>
+                    </p>
+                ) : null}
+                <p className="module-path">
+                    Module Path: <span>{this.props.pluginInfo.modulePath}</span>
+                </p>
+                {this.props.pluginInfo.website ? (
+                    <p className="website">
+                        Website: <span>{this.props.pluginInfo.website}</span>
+                    </p>
+                ) : null}
+                {this.props.pluginInfo.license ? (
+                    <p className="license">
+                        License: <span>{this.props.pluginInfo.license}</span>
+                    </p>
+                ) : null}
+                <p className="capabilities">
+                    Capabilities: <span>{this.props.pluginInfo.capabilities.join(', ')}</span>
+                </p>
+                {this.props.pluginInfo.capabilities.indexOf('webhooker') !== -1 ? (
+                    <p>
+                        Custom Route Prefix:{' '}
+                        {((url) => (
+                            <a
+                                href={url}
+                                target="_blank"
+                                style={{color: 'white'}}
+                                className="custom-route">
+                                {url}
```

Comment:
Can you change this line in the index.tsx
```
const url = slashes.concat(port ? hostname.concat(':', port) : '');
```
Otherwise, the url looks like this: `http://142.1.2.3:/plugin/1/custom/PJSZDRWRvIfTAJC/`